oio/cli/admin/item_repair.py

```diff
@@ -73,6 +75,44 @@ def run(self, parsed_args):
             return 1
 
 
+class AccountRepair(AccountCommandMixin, ItemRepairCommand):
+    """
+    Repair a account by following these steps:
+    recompute the counter of this account ;
+    refresh the counter of all containers in this account.
```

Comment:
The first line of the docstring is used as a short command line help string (for example in the output of `openio-admin --help`). To be formatted properly, it must be on a single line, and followed by an empty line.